content/Watsonx/WatsonxData/202.md

```diff
@@ -0,0 +1,553 @@
+---
+title: '202: Using IBM Cloud and Amazon S3 Object Storage'
+timeToComplete: 90
+updated: 2024-01-29
+---
+<QuizAlert text='Heads Up! Quiz material will be flagged like this!' />
+
+# 202: Using IBM Cloud and Amazon S3 Object Storage
+
+## Goal
+
+The goal of this lab is to integrate IBM watsonx.data with Simple Storage Service (S3) API-compatible object storage hosted by IBM Cloud and Amazon Web Services (AWS).
+
+## Object Storage Overview
+
+Object storage is a type of storage architecture where data is managed as objects. Each object typically includes the data itself, a variable amount of metadata, and a globally unique identifier. As implied by the name, the basic unit of object storage is an object, which are organized in buckets (think of a bucket like a high-level directory or folder). Object storage systems allow retention of massive amounts of structured, semi-structured, and unstructured data, which is well suited for data that is written once and read once or many times. Object storage is low cost, with near unlimited capacity and scalability.
+
+## Prerequisites
+
+### watsonx.data Environment
+
+This lab requires that you already have a provisioned IBM watsonx.data environment. This is done in **[Module 1: Environment Setup](/watsonx/watsonxdata/1)**
+
+### Amazon S3 Object Storage
+
+You must have an Amazon Web Services (AWS) account to be able to use Amazon S3 storage. Create an AWS account and get started with S3 **[here](https://aws.amazon.com/pm/serv-s3/)**
+
+> **Note:** As part of the **[AWS Free Tier](https://aws.amazon.com/free/)**, you can get started with Amazon S3 for free. Upon sign-up, new AWS customers receive 5GB of Amazon S3 storage in the S3 Standard storage class; 20,000 GET Requests; 2,000 PUT, COPY, POST, or LIST Requests; and 100 GB of Data Transfer Out each month.
+
+### IBM Cloud Object Storage (COS)
```

Comment:
Include a section to to let participants know that, in the class room setting, we will provide object storage detail for IBM CLoud and AWS

Reply:
Added a relevant Note